section3/reflection.md

```diff
@@ -1,13 +1,29 @@
 ## Section 3 Reflection
 
 1. In Section 3, we removed even more supports and pushed you to learn a bit more independently, read through technical documentation, and use your resources more. How did that feel?
+- Since I had already done some trouble-shooting in section 2, I felt more comfortable with how I was pushed in section 3. I still feel like I need to read the technical documentation at least twice to get it, but I still feel like I eventually get it.
```

Comment:
That's awesome to hear!

And yes, reading technical docs more than once - that's totally ok. Some will come easier, some you'll need to just take the time and that's part of the job of a dev, so great job practicing that skill 🙌